src/browser/api_protocol/api_handlers/window.js

```diff
@@ -17,11 +17,10 @@ limitations under the License.
 const apiProtocolBase = require('./api_protocol_base');
 const Window = require('../../api/window').Window;
 const Application = require('../../api/application').Application;
-const _ = require('underscore');
 
-let successAck = {
-    success: true
-};
+// some local sugar
+const getWinId = message => apiProtocolBase.getTargetWindowIdentity(message.payload);
+const getGroupingWinId = message => apiProtocolBase.getGroupingWindowIdentity(message.payload);
```

Comment:
prefer braces

Reply:
you mean `x => { return x; }` instead of `x => x` ? 


IMHO that kinda defeats the purpose of
> "An arrow function expression ... has a shorter syntax compared to function expressions ..."
(from [mdn page](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Functions#Arrow_functions)).